src/generator.cpp

```diff
@@ -198,6 +226,15 @@ bool Generator::GeneratorImpl::modifiedProfile() const
                sha1(profileContents) != PYTHON_GENERATOR_PROFILE_SHA1;
 }
 
+std::string Generator::GeneratorImpl::newLineIfNeeded()
```

Comment:
`newLineIfNotEmpty`
```
return mCode.empty() ? {} : "\n";
```

Reply:
Agreed. Done in 6a68a501, although it reads `return mCode.empty() ? "" : "\n"; ` since `{}` cannot be used on the right hand side of `?`.